app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,76 @@
+
+class RentalsController < ApplicationController
+  def checkout
+    customer_id = check_params[:customer_id]
+    movie_id = check_params[:movie_id]
+
+    find_customer_and_movie(customer_id, movie_id)
```

Comment:
I like the idea of moving the shared logic to find the customer and the movie into a helper method. Why not take it a step further and make it a controller filter?